docs/environment.yml

```diff
@@ -28,7 +28,7 @@ dependencies:
   - bqplot
   - ipykernel
   - ipyleaflet
-  - jupyterlite >=0.1.1,<0.2.0
+  - jupyterlite >=0.3.0,<0.4.0
```

Comment:
This should probably be `jupyterlite-core`, since the `jupyterlite` metapackage does not include any kernel anymore: https://jupyterlite.readthedocs.io/en/latest/migration.html#id3

Reply:
And we would need to add a kernel, for example `jupyterlite-pyodide-kernel` or `jupyterlite-xeus`.